hot-reload-analysis/src/test/resources/runtimeInstructionTree/org.jetbrains.compose.reload.analysis.tests.RuntimeInstructionTreeParserTest/test - #152 - composable with eager return/runtime-instructions-tree.txt

```diff
@@ -193,47 +191,44 @@ class TestKt {
                 LabelToken(L59)
                 BlockToken(2)
                 LabelToken(L60)
-                BlockToken(1)
-                SourceInformationMarkerStart(key=ComposeGroupKey(key=-333101486), sourceInformation='C13@270L11:Test.kt')
                 BlockToken(23)
-                SourceInformationMarkerEnd
+                LabelToken(L61)
+                BlockToken(2)
                 LabelToken(L62)
                 BlockToken(2)
+                SourceInformationMarkerEnd
```

Comment:
Wait, I see that the start is removed, but why is there still a 'SourceInformationMarkerEnd'?

Reply:
In this test, the source information markers are added in the process of inlining `Column` function call. We cannot remove these markers markers, because `sourceInformation` flag does not affect them. We only remove these markers from **our** code. 

And all the `SourceInformationMarkerStart`/`SourceInformationMarkerEnd` markers are still consistent (i.e. for each start there is an end)